native/src/main/java/module-info.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2022 WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+module io.ballerina.xlibb.jsonpath {
+    requires io.ballerina.runtime;
+    requires io.ballerina.lang;
+    requires json.path;
+}
```

Comment:
native `build.gradle` file is missing. Could you clone this and validate the build? seems those are not committed

Reply:
I will do it after the PR with base files